components/Op/OpAskFormLong.js

```diff
@@ -195,9 +177,9 @@ class OpAskFormLong extends Component {
       this.props.form.validateFields((err, values) => {
         if (!err) {
           const op = this.props.op
-          const { startDateValue, endDateValue } = this.state
+          const { dateRangeValue } = this.state
           op.date = [] // Dirty work around to not change schema
```

Comment:
This line is not needed any more as you set date to an array in the next line.  It was only needed when we used push.

Reply:
Fixed in https://github.com/voluntarily/vly2/pull/991/commits/b7dceb659582028ddbff88aad5885593db71201c